install_microshift.py

```diff
@@ -0,0 +1,160 @@
+import subprocess
+import time
+import host
+from logger import logger
+
+def microshift_iso_builder(h: host.Host):
+    cmd = f"rm -rf rhocp-4.14.toml fast-datapath.toml"
+    output = h.run(cmd)
+    logger.info(output.out)
+
+    cmd = f"podman rm -f minimal-microshift-server"
+    output = h.run(cmd)
+    logger.info(output.out)
+
+    old_compose = subprocess.check_output(["composer-cli", "compose", "status"])
+    old_compose_splitlines = old_compose.splitlines()
+
+    for line in old_compose_splitlines[1:]:
+        compose_id = line.decode('utf-8').split(" ")[0] 
+        if 'FAILED' or 'FINISHED' not in line.decode('utf-8'):
+            cmd = f" composer-cli compose cancel {compose_id}"
+            output = h.run(cmd)
+            logger.info(output.out)
+    
+        cmd = f"composer-cli compose delete {compose_id}"
+        output = h.run(cmd)
+        logger.info(output.out)
+        
+    cmd = f"dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion firewalld" 
+    output = h.run(cmd)
+    logger.info(output.out)
+    
+    cmd = f"systemctl enable osbuild-composer.socket cockpit.socket --now" 
+    output = h.run(cmd)
+    logger.info(output.out)
+
+    
+    uname_m= subprocess.check_output(['uname', '-m']).decode('utf-8').strip()
+
+    content = f'''id = "rhocp-4.14"\nname = "Red Hat OpenShift Container Platform 4.14 for RHEL 9"\ntype = "yum-baseurl"\nurl = "https://cdn.redhat.com/content/dist/layered/rhel9/{uname_m}/rhocp/4.14/os"\ncheck_gpg = true\ncheck_ssl = true\nsystem = false\nrhsm = true'''
```

Comment:
You can use a file template + Jinja to generate a new file. Might look cleaner.

Reply:
Following up here, I think using  jinja here might obfuscate the file. It's a simple static file with one dynamic parameter that easily retrievable in python. I'm not sure using Jinja will make it cleaner because this is such a simple file. What do you think?

Reply:
Do we plan on being on 4.14 and RHEL 9 forever? Those are some of the extra parameters that may need to be passed in.  I think it is hard to read like a single line that is this long. You can split this line and it might look better.

Reply:
I don't mind having the file coded in line.  We can have:
```
    content = f'''id = "rhocp-4.14"
name = "Red Hat OpenShift Container Platform 4.14 for RHEL 9"
type = "yum-baseurl"
url = "https://cdn.redhat.com/content/dist/layered/rhel9/{uname_m}/rhocp/4.14/os"
check_gpg = true
check_ssl = true
system = false
rhsm = true'''
```

in other words, at least replace the \n by a newline in the file for readability. Next, for the sake of getting this merged, I'm also ok to not have jinja just yet. We can also use `{version}` in place of 4.14 and format it in with the f-string.